nemo/collections/asr/data/audio_to_diar_label.py

```diff
@@ -61,7 +61,7 @@ def get_scale_mapping_list(uniq_timestamps):
     return scale_mapping_argmat
 
 
-def extract_seg_info_from_rttm(uniq_id, rttm_lines, emb_dict=None, target_spks=None):
```

Comment:
you replaced `emb_dict` with `mapping_dict` but I see `emb_dict` variable still at line 84.

Reply:
Also please add doc strings for these variables

Reply:
This is a mistake and I didn't catch it cause i didn't test seq_eval_mode. removed emb_dict. Adding docstrings.